app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        copy, old_file, new_file = command.split()
+
+        if old_file != new_file or copy == "copy":
+            with (
+                open(old_file, "r") as file_int,
+                open(new_file, "w") as file_out
+            ):
+                file_out.write(file_int.read())
+    except Exception:
```

Comment:
you should write more clear exception, what if Exception connected with file , not with command?(for example - file doesn't exist). you should write another message in this case